includes/helper/views/html-licences.php

```diff
@@ -45,6 +45,10 @@
 			?>
 		</div>
 		<?php endif; ?>
+		<form method='post' class='plugin-licence-search'>
```

Comment:
WDYT to use double quotes here? Just to follow the standard of the rest.

Reply:
Sure, I also changed the code to use double quotes in the rest of the HTML too: 9a083b8. 

Thanks!